### PR TITLE
Render multiple legend items on line chart

### PR DESCRIPTION
**Pull request by 64json**

### CODE REVIEW

File: src/components/line-chart-audification/line-chart-audification.component.ts [hunk 1]
```diff
@@ -49,8 +49,13 @@ export class LineChartAudificationComponent implements AudificationPreference, O
...
   }
 
-  get measureNames() {
-    return this.host.measureNames;
+  get legendItems() {
+    return this.host.legendItems;
+  }
+
+  get currentMeasureName() {
+    // TODO: replace 0
```
Comment (jnager) [inline src/components/line-chart-audification/line-chart-audification.component.ts:57]: With what?

File: src/d3/xy-chart.d3.ts [hunk 2]
```diff
@@ -66,31 +104,88 @@ export abstract class XYChartD3 extends BaseD3<RenderOptions> {
...
+    const { legendIconWidth, legendHeight, fontSizeMedium } = XYChartD3;
+
+    this.legendG.html('');
+
+    let offsetX = 0;
+    for (const { label, style } of legendItems) {
+      const itemG = this.legendG
+        .append('g')
+        .attr('transform', `translate(${offsetX}, 0)`);
+      itemG
```
Comment (jnager) [inline src/d3/xy-chart.d3.ts:151]: nit: continue chained operation?

File: src/d3/xy-chart.d3.ts [hunk 3]
```diff
@@ -66,31 +104,88 @@ export abstract class XYChartD3 extends BaseD3<RenderOptions> {
...
+        .append('text')
+        .attr('font-size', fontSizeMedium)
+        .attr('transform', `translate(${textOffsetX}, ${fontSizeMedium})`)
+        .text(label);
+      const textWidth = measureText.node()!.getBBox().width;
+      offsetX += textOffsetX + textWidth + fontSizeMedium;
+    }
+  }
+
+  protected getLegendItemStyler<T extends SVGGraphicsElement>(style: Partial<LegendItemStyle>) {
```
Comment (jnager) [inline src/d3/xy-chart.d3.ts:169]: nit: getLegendItemStyle? Or is Styler intentional?

File: src/components/line-chart-audification/line-chart-audification.component.ts (hunk 1 above)
Comment (64json) [reply src/components/line-chart-audification/line-chart-audification.component.ts:57]: I temporarily made it always get the first legend item, but later it should get the selected measure.
All TODO comments will be resolved by the time I mark this PR non-draft (and ready for review).

File: src/d3/xy-chart.d3.ts (hunk 2 above)
Comment (64json) [reply src/d3/xy-chart.d3.ts:151]: I don't think they should be chained. Line 152 is appending a new `line` element to `itemG`.

File: src/d3/xy-chart.d3.ts (hunk 3 above)
Comment (64json) [reply src/d3/xy-chart.d3.ts:169]: I named it Styler because it returns a function that styles a legend item.

**Pull request merged**